co-simulation/fed/mosaic-carma-messenger/src/main/java/org/eclipse/mosaic/fed/carmamessenger/ambassador/CarmaMessengerRegistrationMessage.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2023 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.eclipse.mosaic.fed.carmamessenger.ambassador;
+
+public class CarmaMessengerRegistrationMessage {
+    private String carmaVehicleId;
+    private String carlaVehicleRole;
+    private String rxMessageIpAddress;
+    private int rxMessagePort;
+    private int rxTimeSyncPort;
+    private String messengerEmergencyState;
+
+    public CarmaMessengerRegistrationMessage(String carmaVehicleId, String carlaVehicleRole, String rxMessageIpAddress,
+            int rxMessagePort, int rxTimeSyncPort, String messengerEmergencyState) {
+        this.carmaVehicleId = carmaVehicleId;
+        this.carlaVehicleRole = carlaVehicleRole;
+        this.rxMessageIpAddress = rxMessageIpAddress;
+        this.rxMessagePort = rxMessagePort;
+        this.rxTimeSyncPort = rxTimeSyncPort;
+        this.messengerEmergencyState = messengerEmergencyState;
```

Comment:
Do we receive the messengerEmergencyState when the CARMA Messenger sends its registration to the CARMA Messenger Ambassador?